nodetypes/http%3A%2F%2Fquantil.org%2Fnodetypes/Qiskit_w1/NodeType.tosca

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="http://quantil.org/nodetypes" id="nodetypes_quantil-Qiskit_w1">
+    <NodeType name="Qiskit_w1" abstract="no" final="no" targetNamespace="http://quantil.org/nodetypes">
```

Comment:
What about the version of Qiskit?

Reply:
Good question: @zimmerml

Reply:
The naming follows the previous implicit conventions. No specified version basically means the last available version.

Reply:
Okay, since we have the versoining, I guess we should add `latest` to the version to be consistent from now on.